tests/std/tests/GH_000431_copy_move_family/test.cpp

```diff
@@ -641,6 +642,85 @@ void test_algorithms(CopyFn copy_fn) {
     }
 }
 
+#if _HAS_CXX20
+struct MyIterator { // A contiguous iterator with a weirdly narrow difference type
+    using iterator_concept = std::contiguous_iterator_tag;
+    using value_type       = int;
+    using difference_type  = short;
+
+    int* ptr = nullptr;

```

Comment:
Suggestion: Everywhere else consistently uses `value_type` and you've made changes in other PRs to be more consistent about using typedefs. It would seem reasonable to replace `int` here too, and in the constructor.